logos/chevron.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 76 76" xmlns="http://www.w3.org/2000/svg"><path d="M33.515 20l17.931 17.726-17.931 17.727L28 49.998l12.416-12.272L28 25.454z"/><path d="M61 0c8.284 0 15 6.716 15 15v46c0 8.284-6.716 15-15 15H15C6.716 76 0 69.284 0 61V15C0 6.716 6.716 0 15 0zm0 6H15a9 9 0 00-9 9v46a9 9 0 009 9h46a9 9 0 009-9V15a9 9 0 00-9-9z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="76" height="76" viewBox="0 0 76 76">
  <polygon fill="currentColor" points="33.515 20 51.446 37.726 33.515 55.453 28 49.998 40.416 37.726 28 25.454"/>
  <path fill="currentColor" d="M61,0 C69.2842712,0 76,6.71572875 76,15 L76,61 C76,69.2842712 69.2842712,76 61,76 L15,76 C6.71572875,76 0,69.2842712 0,61 L0,15 C0,6.71572875 6.71572875,0 15,0 L61,0 Z M61,6 L15,6 C10.0294373,6 6,10.0294373 6,15 L6,61 C6,65.9705627 10.0294373,70 15,70 L61,70 C65.9705627,70 70,65.9705627 70,61 L70,15 C70,10.0294373 65.9705627,6 61,6 Z"/>
</svg>
```

Reply:
* Replaced with less aggressive optimization that might be compatible with those less-compliant programs
* Added `currentColor`.